samples/template-operator/operator/controllers/sample_controller.go

```diff
@@ -104,16 +126,25 @@ func (m *ManifestResolver) Get(obj types.BaseCustomObject) (types.InstallationSp
 			fmt.Errorf("invalid type conversion for %s", client.ObjectKeyFromObject(obj))
 	}
 	return types.InstallationSpec{
-		ChartPath:   chartPath,
+		ChartPath:   m.chartPath,
 		ReleaseName: sample.Spec.ReleaseName,
 		ChartFlags: types.ChartFlags{
 			ConfigFlags: types.Flags{
-				"Namespace":       chartNs,
-				"CreateNamespace": true,
-			},
-			SetFlags: types.Flags{
-				"nameOverride": nameOverride,
+				"Namespace":                chartNs,
+				"CreateNamespace":          true,
+				"DisableOpenAPIValidation": true,
```

Comment:
this will break the `module-chart` aka `redis` scenario. Please make these flags configurable as part of `SetupWithManager` ideally